src/main/java/thewaypointers/trafficsimulator/JunctionLocationTestStarter.java

```diff
@@ -18,9 +13,10 @@
 
     public static void main(String[] args){
 
-        IStateProvider simulation = new JunctionTestProvider();
-        mainFrame=new MainFrame(simulation.getNextState(0));
-        TrafficSimulatorManager.setMainFrame(mainFrame);
-        TrafficSimulatorManager.setSimulation(simulation);
+        TrafficSimulatorManager trafficSimulatorManager = new TrafficSimulatorManager();
+        IStateProvider simulation = new RoadNetworkProvider();
```

Comment:
`JunctionLocationTestStarter` is supposed to use `JunctionTestProvider`, not `RoadNetworkProvider`